dhis-2/dhis-api/src/main/java/org/hisp/dhis/common/EventDataQueryRequest.java

```diff
@@ -112,8 +112,12 @@
 
     private boolean coordinatesOnly;
 
+    private boolean coordinateOuFallback;
+
     private String coordinateField;
 
+    private String fallbackCoordinateField;
```

Comment:
I was wondering, do we need the `fallbackCoordinateField` field? I guess the fallback could be automatic, and we might not need to specify which field to use.

Reply:
Done